release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -2457,6 +2438,38 @@ spec:
 
 * The pipeline metrics API does not support the required `pipelinerun/taskrun` histogram values from {rh-openstack} 1.6 and beyond. Consequently, the *Metrics* tab in the *Pipeline* -> *Details* page is removed instead of displaying incorrect values. There is currently no workaround for this issue. (link: https://bugzilla.redhat.com/show_bug.cgi?id=2074767[*BZ#2074767*])
 
+* In {product-title} {product-version}, pod security admission runs globally with privileged enforcement and restricted audit logging and API warnings. Versions of the `opm` binary released earlier than {product-title} {product-version}, do not work with the restricted profile setting. As a result, an error displays if you install a catalog source that was built with an `opm` binary released earlier than {product-title} {product-version}.
++
+.Example error
+[source,terminal]
+----
+Error: open ./db-xxxx: permission denied
+----
++
+Workaround: Cluster administrators can allow the creation of catalog sources built with earlier `opm` versions by setting the pod security admission labels in the namespace metadata files. Apply the `baseline` label to the `warn`, `audit`, and `enforce` modes and set the `scc.podSecurityLabelSync` label to `false`, as shown in the following example:
++
+.Example `manifests/<target_namespace>.yaml` file
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Namespace
+metadata:
+...
+  labels:
+    security.openshift.io/scc.podSecurityLabelSync: "false"
+    openshift.io/cluster-monitoring: "true"
+    pod-security.kubernetes.io/audit: baseline <1>
+    pod-security.kubernetes.io/warn: baseline <2>
+    pod-security.kubernetes.io/enforce: baseline <3>
+  name: "<namespace_name>"
+----
+<1> Set the `audit` mode label to `baseline`.
+<2> Set the `warn` mode label to `baseline`.
+<3> Set the `enforce` mode label to `baseline`.
```

Comment:
```suggestion
```
@michaelryanpeter the warnings are only warning then do not cause issues and we do not need to suggest them remove/change it as part of a workaround fix. 

Also, we should recommend using the PSA label syncer instead of changing the enforcement so that we let OCP Auth manage the namespace enforce/audit/warn whatever was it is usually does across the cluster